RELEASE_NOTES.md

```diff
@@ -8,6 +8,42 @@ autoconf   | >=2.61
 zlib1g-dev | >=0.13.x
 libtool    | >=2.x
 
-New Features
+This release integrates changes which were merged upstream, but after the **Felmingham v5.1.4**
```

Comment:
I agree, it looks a bit strange. 
I could move it to the relevant place in the list of **Issues Fixed**.